shark/src/main/java/shark/ReferencePattern.kt

```diff
@@ -46,4 +46,10 @@ sealed class ReferencePattern : Serializable {
       return "instance field $className#$fieldName"
     }
   }
+
+  data class NativeGlobalVariablePattern(val className: String) : ReferencePattern() {
+    override fun toString(): String {
+      return "native global variable referencing $className"
```

Comment:
Nit: here and in other classes I think it would be better for readability to go with the expression body:
`override fun toString() = "native global variable referencing $className"`
The strings are short enough to fi within line limit and it will result in nice 3-row data classes

Reply:
Merging, will follow up

Reply:
https://github.com/square/leakcanary/pull/1574